common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -43,6 +44,48 @@ public class CommonValue {
             + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
             + "        }\n"
             + "      }\n";
+    public static final String ML_MODEL_GROUP_INDEX_MAPPING = "{\n" +
+            "  \"_meta\": {\n" +
+            "    \"schema_version\": "+ML_MODEL_GROUP_INDEX_SCHEMA_VERSION+"\n" +
+            "  },\n" +
+            "  \"properties\": {\n" +
+            "    \""+MLModelGroup.MODEL_GROUP_NAME_FIELD+"\": {\n" +
```

Comment:
Missed `backend_roles` field

```
"backend_roles": {
					"type": "text",
					"fields": {
						"keyword": {
							"type": "keyword",
							"ignore_above": 256
						}
					}
				},
```

Reply:
Thanks for the catch. I forgot adding it, I guess